docs/index.bs

```diff
@@ -257,6 +257,14 @@ spec: webappsec-referrer-policy; urlPrefix: https://w3c.github.io/webappsec-refe
         Note: A user agent may use a separate task source for each functional event type in order to avoid a head-of-line blocking phenomenon for certain functional events.
   </section>
 
+  <section>
+    <h3 id="user-agent-bootup">User Agent Boot up</h3>
+
+    A user agent has an associated <dfn export id="dfn-service-worker-manager">service worker manager</dfn>.
+
+    A user agent *must* [=start a new parallel queue=] when it boots up and set the [=service worker manager=] to the result value.
```

Comment:
Does this need to be scoped to the whole user agent, or is per-origin enough?

Reply:
Btw totally happy for this to merge if there's a reason the queue needs to be across the whole browser.

Reply:
This intentionally suggests to be a parallel execution context across the user agent. This matches pretty much what the implementations do (at least Chromium). We can think of the parallel execution context of the parallel queue as a thread in a browser process in Chromium for instance.

Reply:
Is there any reason the registration of a service worker on origin A should block the registration of a service worker on origin B?

Reply:
No. They are independent. Per-origin parallel queues would work and provide better concurrency conceptually. In this change, I just tried to match the current implementation, especially Chromium.

Reply:
I think the spec should allow for maximum concurrency while maintaining the behaviour we want. Implementations are welcome to be less concurrent, but the spec shouldn't reflect this unless developers come to rely on a particular lack of concurrency.

@annevk is that fair?

Reply:
For shared workers I went with a note after requiring a single one for the user agent:

> Each user agent has a single shared worker manager for simplicity. Implementations could use one per origin; that would not be observably different and enables more concurrency.

Reply:
Implementers' feedback would also be great.

/cc @mattto, @wanderview, @mkruisselbrink, @aliams, @hober